src/Microsoft.AspNetCore.SystemWebAdapters/Ref.Standard.cs

```diff
@@ -448,6 +448,106 @@ public enum SameSiteMode
         None = 0,
         Strict = 2,
     }
+
+    /// <summary>
+    /// Provides utility methods for common virtual path operations.
+    /// </summary>
+    public static class VirtualPathUtility
```

Comment:
Can you run the `./update-apis.ps1` script? This file is autogenerated (along with the type forwards)

Reply:
I did run that, but it seemed to overwrite other customizations to those files, so I just made the changes manually.
I didn't want to mix up changes for VPU with other stuff.
TypeForwards.Framework:

- Loses the comments and licence info
- Adds type forwards for Microsoft.AspNetCore.SystemWebAdapters.ISessionKeySerializer, Microsoft.AspNetCore.SystemWebAdapters.ISessionState and Microsoft.AspNetCore.SystemWebAdapters.SessionState.Serialization.ISessionSerializer
- Loses the #if NET472_OR_GREATER for System.Web.SameSiteMode

Ref.Standard:

- Adds implicit operators to HttpRequestBase for Microsoft.AspNetCore.Http.HttpRequest and HttpResponseBase for Microsoft.AspNetCore.Http.HttpResponse

Also, I would point out that the error message is slightly oddly worded: "Only _support_ when running on ASP.NET Core or System.Web" which is why I used a const with "Only _supported_ when running on ASP.NET Core or System.Web"

I noticed that the netstandard/ref version doesn't include the doc comments, so you don't get intellisense in class libraries targeting netstandard2.0 hence why I kept the comments in too.

I think the only other difference between my manual changes and the script is making the ref class partial, but I don't see that actually being used for anything.

Let me know how you want me to proceed.

Reply:
Good catch - I'd been the main person using it so I got used to the workarounds. I've added #181 to fix it up so it should work as expected (and the build will run it to to ensure we didn't miss anything).

> Also, I would point out that the error message is slightly oddly worded:
Yeah, I kept forgetting to fix that. Thanks for point it out - it's fixed in #181

> I noticed that the netstandard/ref version doesn't include the doc comments
I hadn't realized that! Thanks. I'd like to figure out a fix that brings that over for all of them. I'll create a separate issue to track that

> Let me know how you want me to proceed.
For now, it's fine, but once #181 is merged, please merge it in and run the script so things will stay in sync

Reply:
#181 is merged - please merge it in and run the script

Reply:
Hi - found a problem with the result of the script.
It brought the attribute [System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute("virtualPath")] through for `AppendTrailingSlash` - however this doesn't exist on netstandard2.0.
I can remove the attribute from the method if required? - and it is the only method to which this actually applies.

Reply:
If you pull in this commit it should work: https://github.com/twsouthwick/systemweb-adapters/commit/f166b7c4d45db18acfde280ebe25dd565694512f

Reply:
I have put in a manual work-around for now - but it feels like code smell 🫤

Reply:
That commit I referenced is the non-hacky way :)

Reply:
Much better! I'm just struggling with git to try and work out how to pull in your commit to my fork!?

Reply:
If you add my fork as a remote and fetch the changes, you should be able to then do `git cherry-pick f166b7c`

Reply:
Cool. I think that should be that added now and reverted my hacky nastiness.